lib/ajv.d.ts

```diff
@@ -92,7 +92,8 @@ declare namespace ajv {
       data: any,
       dataPath?: string,
       parentData?: Object | Array<any>,
-      parentDataProperty?: string | number
+      parentDataProperty?: string | number,
+      rootData?: any
```

Comment:
Thanks @epoberezkin - I made that change.  You can squash the 2 commits if you want before you merge. I was guessing it had the same type as `data` because I saw https://github.com/epoberezkin/ajv/blob/master/lib/dot/validate.jst#L51 ...

Should data be updated so it is `Object | Array<any>` too?